CMakeLists.txt

```diff
@@ -307,7 +307,8 @@ if ( ENABLE_CPACK )
                            "ipython-notebook,"
                            "python-matplotlib,"
                            "python-scipy,"
-                           "libpocofoundation11,libpocoutil11,libpoconet11,libpoconetssl11,libpococrypto11,libpocoxml11" )
+                           "libpocofoundation11,libpocoutil11,libpoconet11,libpoconetssl11,libpococrypto11,libpocoxml11",
```

Comment:
I'm getting a CMake warning:
```CMake Warning (dev) in CMakeLists.txt:
  Syntax Warning in cmake code at

    /mnt/data1/source/github/mantidproject/mantid/CMakeLists.txt:310:121
```
I think the comma at the end of the line (310) needs to be inside the quotes as the string is one long string concatenated by cmake.